src/liteclient.cc

```diff
@@ -278,6 +278,19 @@ void LiteClient::writeCurrentTarget(const Uptane::Target& t) const {
   if (!tmp.empty()) {
     ss << "CONTAINERS_SHA=\"" << tmp << "\"\n";
   }
+
+  ss << "TAG=\"";
+  auto tags = custom["tags"];
```

Comment:
`auto&` or `const auto&` might save some room in a stack :), not critical at all.